### PR TITLE
Add RHEL 9 mockbuild

### PR DESCRIPTION
**Pull request by ondrejbudai**
Let's start the weird journey of adding RHEL 9 support. This PR adds support for mockbuild.

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] create a file in [news/unreleased](https://github.com/osbuild/osbuild-composer/tree/main/docs/news/unreleased) directory if this change should be mentioned in the release news
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: schutzbot/mockbuild.sh [hunk 1]
```diff
@@ -42,13 +42,19 @@ if curl --silent --fail --head --output /dev/null "${REPO_URL}/repodata/repomd.x
...
     curl -Ls --retry 5 --output /tmp/epel.rpm \
         https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
     sudo rpm -Uvh /tmp/epel.rpm
+elif [[ $ID == rhel || $ID == centos ]] && [[ ${VERSION_ID%.*} == 9 ]]; then
+    # we have our own small epel for EL9, let's install it
+
+    # install Red Hat certificate, otherwise dnf copr fails
+    curl -LO --insecure https://hdn.corp.redhat.com/rhel8-csb/RPMS/noarch/redhat-internal-cert-install-0.1-23.el7.csb.noarch.rpm
+    sudo dnf install -y ./redhat-internal-cert-install-0.1-23.el7.csb.noarch.rpm dnf-plugins-core
+    sudo dnf copr enable -y copr.devel.redhat.com/osbuild-team/epel-el9 rhel-9.dev-x86_64
```
Comment (atodorov) [inline schutzbot/mockbuild.sh:57]: `x86_64` hard-coded at the end while you've got aarch64 in test runners.

File: schutzbot/terraform [hunk 2]
```diff
@@ -1 +1 @@
-3afb50b0f1b4e7476e03151fc8efa81d29ffde49
+429f0e870d9967ab76049c337ee2ee432433aaba
```
Comment (atodorov) [inline schutzbot/terraform:1]: FTR this is going to conflict with #1573!

File: schutzbot/rhel-9-mock-configs/templates/rhel-9.tpl [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+[rhel9-crb]
+name=RHEL 9 CRB
+baseurl=http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9-Beta/latest-RHEL-9/compose/CRB/$basearch/os/
+enabled=1
+gpgcheck=0
+
+[rhel9-buildroot]
+name=RHEL 9 Buildroot
+baseurl=http://download.eng.bos.redhat.com/rhel-9/nightly/BUILDROOT-9-Beta/latest-BUILDROOT-9/compose/Buildroot/$basearch/os
```
Comment (atodorov) [inline schutzbot/rhel-9-mock-configs/templates/rhel-9.tpl:49]: Isn't this information in test/data/repositories/rhel-90.json already? I don't understand why we need to define these URLs twice and why they are different between the two files.

**Review by atodorov — CHANGES_REQUESTED** [review]
Some changes / questions requested.

File: schutzbot/mockbuild.sh (hunk 1 above)
Comment (ondrejbudai) [reply schutzbot/mockbuild.sh]: Thanks!

File: schutzbot/rhel-9-mock-configs/templates/rhel-9.tpl (hunk 3 above)
Comment (ondrejbudai) [reply schutzbot/rhel-9-mock-configs/templates/rhel-9.tpl:51]: No, repositories for image building include only BaseOS and AppStream whereas for building RPMs we also need CRB and buildroot.

**Comment by ondrejbudai** [comment]
This should be now fully functioning and ready to be merged.

**Review by teg — APPROVED** [review]

**Review by atodorov — APPROVED** [review]

**Pull request merged**